elide-spring/elide-spring-boot-autoconfigure/src/test/java/example/tests/AsyncTest.java

```diff
@@ -82,59 +81,34 @@ public void testAsyncApiEndpoint() throws InterruptedException {
                         .body("data.id", equalTo("ba31ca4e-ed8f-4be0-a0f3-12088fa9263d"))
                         .body("data.type", equalTo("asyncQuery"))
                         .body("data.attributes.queryType", equalTo("JSONAPI_V1_0"))
-                        .body("data.attributes.status", equalTo("COMPLETE"))
-                        .body("data.relationships.result.data.type", equalTo("asyncQueryResult"))
-                        .body("data.relationships.result.data.id", equalTo("ba31ca4e-ed8f-4be0-a0f3-12088fa9263d"));
+                        .body("data.attributes.status", equalTo("COMPLETE"));
 
                 // Validate AsyncQueryResult Response
                 given()
                         .accept("application/vnd.api+json")
-                        .get("/json/asyncQuery/ba31ca4e-ed8f-4be0-a0f3-12088fa9263d/result")
+                        .get("/json/asyncQuery/ba31ca4e-ed8f-4be0-a0f3-12088fa9263d")
```

Comment:
There is no need to issue a separate request here.

Reply:
removed extra requests